scripts/storybook/runner.js

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+
+/**
+ * This is a start-storybook wrapper that automatically pre-builds dependencies that are needed to run storybook,
+ * then invokes `start-storybook` so CLI experience is the same as running raw `start-storybook` binary
+ *
+ *
+ * Note:
+ * - This is used only for vNext packages
+ * - We cannot leverage build-less experience unfortunately at the moment. To have it we will need a bigger overhaul of "how we build" things.
+ *
+ * @example
+ *
+ * ```sh
+ * node ./scripts/storybook/runner
+ * ```
+ */
+
+const path = require('path');
+const fs = require('fs');
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+
+main();
+
+function main() {
+  const args = process.argv.slice(2);
+  const COMMAND_PREFIX = `${chalk.cyan('>')} ${chalk.inverse(chalk.bold(chalk.cyan(' STORYBOOK RUNNER ')))}`;
+  const implicitDependencies = [
+    {
+      name: '@fluentui/react-storybook-addon',
+      description: 'fluentui storybook addon that adds functionality to storybook',
+    },
+  ];
+  const dependencies = [
+    { name: '@fluentui/babel-make-styles', description: 'custom babel plugin that compiles make-styles definitions' },
+  ];
+
+  /**
+   * @type {{name:string;devDependencies: Record<string,string>,[key:string]:unknown}}
+   */
+  const projectPackageJson = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'package.json'), 'utf-8'));
+
+  const dependenciesToBuild = [
+    ...dependencies.filter(dep => {
+      return projectPackageJson.devDependencies[dep.name];
+    }),
+    ...implicitDependencies,
+  ];
+
+  const shouldInvokeHelp = args.includes('--help');
+  const shouldExecPreBuild = !shouldInvokeHelp && dependenciesToBuild.length > 0;
+
+  const storybookCommand = `start-storybook ${args.join(' ')}`;
+  const dependencyBuildCommand = `lage build --to ${dependenciesToBuild.map(dep => dep.name).join(' ')}`;
+
+  printTitle();
+
+  if (shouldExecPreBuild) {
+    printDependencies();
+
+    execSync(dependencyBuildCommand, { stdio: [0, 1, 2] });
```

Comment:
In the future can you just use 'inherit' instead of 0 1 2? It's easier to understand at a glance, and 0 1 2 is hard to look up if you're not already familiar with what it means.

Reply:
hmm so from my POV none of these options is "easy to understand" as one needs to go to docs most of the time anyways. based on that there is not much difference what is being used -> https://nodejs.org/api/child_process.html#child_process_options_stdio

Unless we have lint rule or proper styled guide for how to handle theses things, this is not a very good time investment from my experience. also from quick search in the project there are already multiple inconsistencies.

Reply:
Thanks for making a PR with the update! I think it was a reasonable request because the word "inherit" has meaning, which makes it easier to be reminded or take a guess at what it means in this context, and it's easier to look up if needed. It's also what nearly everywhere else in the repo uses even without a lint rule (either `inherit` or `pipe`, except two places that pass an array for more customization).